tree/ntuple/doc/BinaryFormatSpecification.md

```diff
@@ -684,6 +685,64 @@ The entry range allows for finding the right page list for random access request
 The number of clusters information allows for using consistent cluster IDs
 even if cluster groups are accessed non-sequentially.
 
+#### Linked Attribute Sets
+
+An RNTuple may have zero or more linked Attribute Sets, containing metadata.
+Each Attribute Set is stored on disk as an RNTuple and the Anchor of each RNTuple is linked to by the main
+RNTuple's footer.
+
+An Attribute Set RNTuple has a number of restrictions compared to a regular RNTuple:
+
+1. it cannot have linked Attribute RNTuples itself;
+2. the Alias columns sections, both in its header and footer, must be empty (i.e. none of the Attribute Set RNTuple's
+   Fields can be Projected Fields);
+3. none of its fields may have a structural role of 0x04 (i.e. it must not contain a ROOT streamer object);
```

Comment:
For consideration, maybe move into a separate top-level section (between Envelopes and Mappings) explaining Attribute Sets rather than nested inside the Footer Envelope?

Reply:
Hopefully I got it right this time!